website/pages/en/support.js

```diff
@@ -16,7 +16,8 @@ The React Native for Windows (RNW) Team strives to provide full support for the
 | Version | Support Phase | Release Date | Active Support Start | Maintenance Support Start | End of Support |
 | -- | -- | -- | -- | -- | -- |
 | [main](https://www.npmjs.com/package/react-native-windows/v/canary) | [Canary](#canary-support) | *N/A* | *N/A* | *N/A* | *N/A* |
-| [0.73](https://www.npmjs.com/package/react-native-windows/v/latest) | [Active](#active-support) | 12/11/2023 | 12/11/2023 | *TBD* | *TBD* |
+| [0.74](https://www.npmjs.com/package/react-native-windows/v/v0.72-stable) | [Active](#active-support) | 4/29/2024 | 4/29/2024 | *TBD* | *TBD* |
+| [0.73](https://www.npmjs.com/package/react-native-windows/v/latest) | [Unsupported](#unsupported) | 12/11/2023 | 12/11/2023 | 4/29/2024 | 7/29/2024 |
```

Comment:
@jonthysell What's the right defaults for support window here?

Reply:
Followed our documentation where we support for two months in maintenance after a release! fixed the npm tag though